takoyaki/src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +12,11 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static final class kIntake {
+        public static final int FALCON_MOTOR_ID = -1;
+        public static final int SOLENOID_FRONT = 1;
+        public static final int SOLENOID_BACK = 0;
+        public static final float INTAKE_SPEED = 0.9f;
+    }
```

Comment:
Manav came up with the idea of adding driver control buttons to constants. I think that's the way to go, and you should add it. That'll make it easier to do things like single controller mode later